docs/docs/docs/01-core/content-releases/02-frontend/01-list-view.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Releases list view
+tags:
+  - content-releases
+  - tech design
+---
+
+### Overview
+
+The list-view provides a comprehensive display of all available content releases. Organized into two tabs, it efficiently categorizes content into pending releases (those not yet published) and completed releases (already published).
+
+#### New Release Creation:
+
+The creation of a new release is made simple with the inclusion of a dedicated button. When activated, this button opens a modal housing a user-friendly form. Importantly, access to this creation feature is contingent on the user holding the necessary permission, specifically `plugin::content-releases.create` otherwise the button is disabled. And for validation the release name needs to be unique and not empty.
```

Comment:
```suggestion
If a user has the permissions to create a release a 'New release' button will be present in the header. Clicking the button opens a form requiring a unique release name. 
```

I believe the button is hidden when permissions are not granted, but I could be wrong...